presto-parser/src/main/java/com/facebook/presto/sql/tree/ShowGrants.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class ShowGrants
+    extends Statement
+{
+    private final boolean table;
+    private final Optional<QualifiedName> tableName;
+    private final boolean all;
+
+    public ShowGrants(boolean table, Optional<QualifiedName> tableName, boolean all)
```

Comment:
We we add support for grants on functions and other things, we will ned a way to differentiate them.